Fix typos, rephrase words and improve readability (#813)
shopware/docs: guides/installation/flex.md

composer create-project shopware/production:dev-flex <project-name>
```

This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed once the template is generally available.

> The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront` and `shopware/elasticsearch`. If you don't need any of them, you can uninstall it with:

Reviewer: ```suggestion
The template contains all Shopware bundles like `shopware/administration`, `shopware/storefront` and `shopware/elasticsearch`. If you don't need any, then you can uninstall them with:
```

composer create-project shopware/production:dev-flex <project-name>
```

> This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed when the template is generally available.

Author: ```suggestion
This creates a new project in the `<project-name>` directory. The `dev-flex` version constraint installs the latest version of Shopware. The constraint `dev-flex` will be removed once the template is generally available.
```
